### DISCUSSION TITLE
Blank canvas

### DISCUSSION THREAD

**Discussion by JDMeijers** [cssbattle/forum#231] [Bugs reports]
All of a sudden (since an hour or so) on some targets I see a blank canvas (exam. target 71 top solution). On some targets the rendering is quite random (mostly on targets where a variable is used). This happens to me on Edge & Chrome browser on Windows 10. I am not seeing any errors or anything.

Thanks in advance!

**Comment by chinchang** [comment]
Weird! Works fine for me on Target 71 and other targets. Even with variables. Can you also specify your browser versions...may be that can help

**Reply by JDMeijers** [reply]
Edge Version 99.0.1150.36 (Official build) (64-bit)
Chrome Version 99.0.4844.51 (Official build) (64-bit)

**Reply by JDMeijers** [reply]
![Capture](https://user-images.githubusercontent.com/51381453/157253353-26d1814f-57ed-47f2-9a85-c83b2de6ebd6.PNG)

**Reply by jorgeluis279hch** [reply]
Hola, me sucede los mismo estoy en la versión de Chrome 99.0.4844.51
![bg](https://user-images.githubusercontent.com/35770157/157313275-3c87d2ea-4d6e-4ca0-a47e-aa0b1145acb6.png)

**Comment by himanshubhatt1** [comment]
Same problem is happening with me too...🤨

**Comment by chinchang** [comment]
Update: Able to replicate in latest chrome 99. Seems like something changed related to parsing of unformatted CSS. We are checking. Appreciate any hints here. 🙏🏼

**Reply by chinchang** [reply]
Quick observation: `--` doesn't look like a valid variable anymore! 🧐 Digging more...

**Reply by rafehimself** [reply]
it gets 100%, however previewer doesn't work
show empty white screen with `var(--)`
![image](https://user-images.githubusercontent.com/58576132/157734648-c4b53baa-e131-4bcb-9137-25bd10b9c299.png)

a problem since many results use that

**Reply by SelenIT** [reply]
It was a change in the CSS standard itself. The name `--` was valid for the custom property at the time, but is not anymore. Eventually these solutions will stop working on the backend too. There is already a lot of solutions that don’t work anymore after backend update, I believe it’s inevitable.

**Reply by chinchang** [reply]
Thanks @SelenIT for that update! 🙏🏼

**Reply by siva-kannan3** [reply]
<img width="1010" alt="Screenshot 2022-03-13 at 12 28 10 AM" src="https://user-images.githubusercontent.com/60533560/158031249-44b0f6d7-08a6-4f7f-9ce7-df5dc52775cd.png">

https://www.w3.org/TR/css-variables-1/

**Comment by w1ll3m** [comment]
The short -- variable stopped working in the browser, but empty variables are now working.
So the empty --b in `*{border-radius:0 var(--b,50px)}img{background:red;padding:50;--b:}` will remove the img border radius in the preview in chrome now. This does not work in the backend renderer however!

**Reply by SelenIT** [reply]
Thanks for mentioning this! It will work after the next backend update, so nice to remember!
